### PR TITLE
feat: add pre-built binaries for Node 12

### PR DESCRIPTION
**Pull request by nolanmar511**
Part of #66 

After this, I'll need to enable continuous and presubmit unit tests for Node 12 (though I add the kokoro configs here).

### CODE REVIEW

**Comment by kalyanac** [comment]
`gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
`

**Comment by nolanmar511** [comment]
PTAL

E2E tests are now passing (I'd forgotten to update the script so binaries for 12 were actually built).

File: system-test/Dockerfile.node12-alpine [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN go get github.com/google/pprof
+
+
+FROM node:12-alpine
+
+ARG ADDITIONAL_PACKAGES
+
+RUN apk add --no-cache bash $ADDITIONAL_PACKAGES
+WORKDIR /root/
+COPY --from=0 /go/bin/pprof /bin
```
Comment (kalyanac) [inline system-test/Dockerfile.node12-alpine]: `--from=builder`

Comment (nolanmar511) [reply system-test/Dockerfile.node12-alpine]: Done (for this and other alpine docker files)

**Comment by nolanmar511** [comment]
PTAL

**Pull request merged**